sockeye/inference.py

```diff
@@ -551,13 +552,16 @@ def get_max_output_length(input_length: int):
 :param tokens: List of translated tokens.
 :param attention_matrix: Attention matrix. Shape: (target_length, source_length).
 :param score: Negative log probability of generated translation.
+:param beam_history: List of beam histories. The list will contian more than one
+history if it was split due to exceeding max_length.
 """
 
 TokenIds = List[int]
 Translation = NamedTuple('Translation', [
```

Comment:
Can we make this a proper class to have direct typing in the constructor?
Also we could type `beam_history` to `Optional[List[Dict[str, List]]]`, or maybe make the inner dict a class itself.

Reply:
I think Matt was turning some of these namedtuples into classes in his PR, but this one was kept. Is it still a concern that it's not a class?

Reply:
I think its fine leaving Translation as a NamedTuple for now. But I think typing the beam history or making it a proper class would be a good idea. 
Also, there is a typo in the docstring: contian -> contain

Reply:
On a second thought ( I didn't see that you pushed an update to this PR): I think it would be easier to make this a proper class because then you can set beam_histories in the constructor to `None` by default and you dont have to pass it everywhere in the tests and in the rest of the code.